instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/jaeger/JaegerExporterAutoConfiguration.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.exporters.jaeger;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.opentelemetry.exporters.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.instrumentation.spring.autoconfigure.TracerAutoConfiguration;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/** Create JaegerExporter */
+@Configuration
+@EnableConfigurationProperties(JaegerExporterProperties.class)
+@AutoConfigureBefore(TracerAutoConfiguration.class)
+@ConditionalOnProperty(prefix = "opentelemetry.trace.exporter.jaeger", name = "enabled")
+public class JaegerExporterAutoConfiguration {
+
+  @Bean
+  @ConditionalOnMissingBean
+  public JaegerGrpcSpanExporter otelJaegerExporter(JaegerExporterProperties jaegerExporterProperties) {
+
+    ManagedChannel channel =
+        ManagedChannelBuilder.forAddress(
+                jaegerExporterProperties.getHost(), jaegerExporterProperties.getPort())
+            .usePlaintext()
+            .build();
+
+    return JaegerGrpcSpanExporter.newBuilder()
```

Comment:
which artifact does the exporter live in? i.e., which artifact do we need to extend?

Reply:
spring auto-configurations can simply detect that JaegerGrpcSpanExporter is in the classpath, e.g., `@ConditionalOnClass`, and on by default, unless turned off.

Reply:
The jaeger exporter lives in the opentelemetry-java/exporters directory. The artifact is io.opentelemetry:opentelemetry-exporters-jaeger. My plan was to supply the four exporter artifacts (OTLP, JAEGER, ZipKin, LoggingSpan) as apis. These exporters would only be set if the field `opentelemetry.trace.exporter.exporterName.enabled=True` and if the bean is missing. 

I think a better approach would be to create starters for each exporter and using @ConditionalOnClass. I'll make this change